.travis.yml

```diff
@@ -34,6 +34,7 @@ before_install:
   - ./miniconda.sh -b -p ~/mc
   - export PATH=~/mc/bin:$PATH
   - conda config --set always_yes yes --set changeps1 no --set quiet true
+  - conda config --append channels nsls2forge
```

Comment:
Honestly someone should just fix the databroker recipe.

Reply:
They moved all their packages to the nsls2forge channel.